modules/cnf-understanding-workload-hints.adoc

```diff
@@ -60,4 +61,9 @@ perPodPowerManagement: true
     | Critical and non-critical workloads
     | Allows for power management per pod.
 
-|===
\ No newline at end of file
+|===
++
+[NOTE]
```

Comment:
[NOTE] block is empty. Did you mean to remove it?

Reply:
Yes I meant to remove my bad.